### PR TITLE
Bump dependencies (step 1)

### PR DESCRIPTION
**Pull request by lpellegr**
Some dependencies have not been upgraded since 5 years or more. The idea of this pull request is to upgrade as much dependencies as possible in order to take advantage of:

  - performance improvements
  - bug fixes
  - security fixes
  - new available features that could ease future developments

### CODE REVIEW

**Comment by lpellegr** [comment]
@youribonnaffe The work for this pull-request is not yet finished. However, I would really appreciate your comments on existing commits if you have time before leaving.

**Comment by youribonnaffe** [comment]
retest this please

**Comment by youribonnaffe** [comment]
:+1: :+1000: :)

**Comment by lpellegr** [comment]
Retest this please.

**Comment by lpellegr** [comment]
@ow2-proactive/developers Could two people please review the changes?

**Comment by paraita** [comment]
It's OK for me

File: rest/rest-server/src/test/java/functionaltests/RestSchedulerPushPullFileTest.java [hunk 1]
```diff
@@ -208,8 +231,10 @@ public void testIt(String spaceName, String spacePath, File destPath, boolean en
         HttpResponse response4 = executeUriRequest(reqDelete);
 
         System.out.println(response4.getStatusLine());
+
         assertHttpStatusOK(response4);
 
-        Assert.assertTrue(destFile + " does not exist", !destFile.exists());
+        Assert.assertTrue(destFile + " still exist", !destFile.exists());
```
Comment (marcocast) [inline rest/rest-server/src/test/java/functionaltests/RestSchedulerPushPullFileTest.java]: shouldn't be does not exist?

Comment (lpellegr) [inline rest/rest-server/src/test/java/functionaltests/RestSchedulerPushPullFileTest.java]: The message is displayed when the assertion is `false`. So I expect to see a message saying why the assertion was wrong not the reverse of why the assertion is `false`. The javadoc seems not to say a lot about how the message should be written:  

```
     * Asserts that a condition is true. If it isn't it throws an
     * {@link AssertionError} with the given message.
     *
     * @param message the identifying message for the {@link AssertionError} (<code>null</code>
     * okay)
     * @param condition condition to be checked
```

**Comment by marcocast** [comment]
looks fine to me

**Pull request merged**